locales/de-DE/translation.json

```diff
@@ -148,57 +148,57 @@
-  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "Alephium hat keinen Zugriff auf Ihre Brieftasche.\nSie sind der einzige Besitzer.",
+  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "Alephium hat keinen Zugriff auf Ihre Wallet. \nSie sind der einzige Besitzer.",
   "Back": "Zurück",
   "Type your recovery phrase": "Geben Sie Ihre Wiederherstellungsphrase ein",
-  "Error while importing wallet": "Fehler beim Importieren der Brieftasche",
-  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Bewahren Sie die Worte an einem sicheren Ort auf! Sie sind die geheime Wiederherstellungsphrase Ihrer Brieftasche.",
+  "Error while importing wallet": "Fehler beim Importieren der Wallet",
+  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Bewahren Sie die Worte an einem sicheren Ort auf! Sie sind die geheime Wiederherstellungsphrase Ihrer Wallet.",
   "All good? Let's continue!": "Alles in Ordnung? Machen wir weiter!",
   "Everything is ready!": "Alles ist bereit!",
-  "Welcome to Alephium.": "Willkommen in Alephium.",
+  "Welcome to Alephium.": "Willkommen bei Alephium.",
   "Let's go!": "Los geht's!",
   "welcomeScreenOneAddressPerGroupMessage": "Fortgeschrittener Benutzer: Möchten Sie mit <1>einer Adresse pro Gruppe für Mining oder DeFi beginnen?</1>",
   "Show advanced options": "Erweiterte Optionen anzeigen",
   "Hide advanced options": "Erweiterte Optionen ausblenden",
-  "For mining or DeFi use.": "Für Bergbau oder DeFi verwenden.",
+  "For mining or DeFi use.": "Für Mining oder DeFi verwenden.",
   "Address copied to clipboard!": "Adresse in die Zwischenablage kopiert!",
-  "Your Wallet": "Ihre Brieftasche",
+  "Your Wallet": "Ihre Wallet",
   "Your address": "Ihre Adresse",
-  "I've copied the words, continue": "Ich habe die Worte kopiert, fahren Sie fort",
-  "Advanced operations reserved to more experienced users. A \"normal\" user should not need to use them very often, if not at all.": "Fortgeschrittene Operationen, die erfahreneren Benutzern vorbehalten sind. Ein \"normaler\" Benutzer sollte sie nicht sehr oft, wenn überhaupt, verwenden müssen.",
-  "It appears that your wallet has too many UTXOs to be able to send this transaction. Please, consolidate (merge) your UTXOs first. This will cost a small fee.": "Es scheint, dass Ihre Brieftasche zu viele UTXOs hat, um diese Transaktion senden zu können. Bitte konsolidieren (fusionieren) Sie zunächst Ihre UTXOs. Dies wird eine kleine Gebühr kosten.",
+  "I've copied the words, continue": "Ich habe die Worte kopiert, weiter",
+  "Advanced operations reserved to more experienced users. A \"normal\" user should not need to use them very often, if not at all.": "Fortgeschrittene Tätigkeit die erfahreneren Benutzern vorbehalten sind. Ein \"normaler\" Benutzer sollte sie nicht sehr oft, wenn überhaupt, verwenden müssen.",
```

Comment:
I would translate it to "Fortgeschrittene Funktionen" because later there is a "passphraseWarningMessage" that is using also "fortgeschrittene Funktion"